packages/docs/operators/_build.yaml

```diff
@@ -0,0 +1,166 @@
+# Copyright 2020-2022 Lowdefy, Inc
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+_ref:
+  path: templates/operators.yaml.njk
+  transformer: templates/operatorsMethodTransformer.js
+  vars:
+    pageId: _build
+    pageTitle: _build
+    filePath: operators/_build.yaml
+    description: |
+      The `_build` operator evaluates at build time. This can be used to loop over a list in a JSON file generated by another application to calculation a value a build time or to read environment variables.
+    methods:
+      - name: env
+        types: |
+          ```
+          (key: string): any
+          (all: boolean): any
+          ```
+        description: |
+          The `_build.env` method reads from your environment variables. The return value of `_build.env` will form part of the build output, so the [`_secret`](/_secret) operator should rather be used for reading any secrets.
+        examples: |
+          ###### Read environment from environment variables:
+          With the following development and production environment variables, we can use the `_build.env` method to determine which environment we are in.
+
+          `Development environment .env`
+          ```
+          MY_ENVIRONMENT="Development"
```

Comment:
Environment vars are normally lowercase, so I think this should rather be `production` and `development`

Reply:
Changes made in [bafc31d](https://github.com/lowdefy/lowdefy/pull/1469/commits/bafc31d6dde5558f5ef0025955a7cb9b326f3f06)